src/views/Graph.vue

```diff
@@ -361,6 +379,13 @@ export default {
               icon: mdiArrowCollapse,
               action: 'callback',
               callback: this.decreaseSpacing
+            },
+            {
+              title: 'Organise by cycle point',
```

Comment:
```suggestion
              title: 'Group by cycle point',
```

Reply:
I used this phrasing as its what is used in Cylc 7 but can change it